### DISCUSSION TITLE
"Keine Antwort" Status lightbulb

### DISCUSSION THREAD

**Discussion by mhoster83** [jensweigele/ioBroker.yahka#300] [Q&A]
Hallo Community,
hat jemand einen Tipp für mich, welchen Custom Charachteristic ich wie konfigurieren muss, damit ich bei meinen LightBulbs den Status "Nicht erreichbar" ausgeben kann, wenn diese nicht erreichbar sind?
Mein Ziel ist es, dass der Status so wie in dem beigefügten Bild erscheint.
Vielen Dank!
[image](https://user-images.githubusercontent.com/70483786/137582487-d93534c9-1da7-4421-a57b-f174076da03d.jpeg)

**Comment by jensweigele** [comment]
Dieses Feature wird von Homekit nicht unterstützt. Siehe auch: https://github.com/jensweigele/ioBroker.yahka/issues/290

**Reply by mhoster83** [reply]
Danke
